.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"      # Location of your workflow files
+    schedule:
+      interval: "weekly"  # Options: daily, weekly, monthly
```

Comment:
Can we monthly? The repo isn't super active and I have so many bot PRs to review these days. 😅

Reply:
For sure, definitely want the update.